src/System.Private.ServiceModel/tests/Scenarios/Bridge/Bridge/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Bridge")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Bridge")]
+[assembly: AssemblyCopyright("Copyright ©  2015")]
```

Comment:
The build targets generate AssemblyInfo's for the other projects.  I don't know what is required for Bridge to tap into those targets (look for versioning.targets after restoring build tools).  We can file an issue and revisit this after checkin